clang/test/Frontend/cir-not-built.c

```diff
@@ -0,0 +1,12 @@
+// Test that using -emit-cir when clang is not built with CIR support gives a proper error message
+// instead of crashing.
+
+// This test should only run when CIR support is NOT enabled
+// REQUIRES: !cir-support
```

Comment:
Have you considered
```suggestion
// UNSUPPORTED: cir-support
```
?